python/servo/bootstrap.py

```diff
@@ -15,7 +15,29 @@
 from servo.util import extract, download_file, host_triple
 
 
-def salt(context, force=False):
+def run_as_root(command):
+    if os.geteuid() != 0:
+        try:
+            # Check if sudo'ing possible
+            subprocess.check_call(['sudo', '-v'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+            command.insert(0, 'sudo')
+        except:
+            command = ['su', 'root', '-c', ' '.join(command)]
+
+    return subprocess.call(command)
+
+
+def install_salt_dependencies(context, force):
```

Comment:
This method should use the `force` parameter as it is used elsewhere: if `force` is False, then it should prompt the user before installing packages.